lib/Wikia/src/Util/GlobalStateWrapper.php

```diff
@@ -60,7 +60,7 @@ public function wrap( $function ) {
 	protected function captureState() {
 		$this->capturedState = array();
 		foreach ( $this->globalsToWrap as $globalKey => $globalValue ) {
-			$this->capturedState[$globalKey] = $GLOBALS[$globalKey];
+			$this->capturedState[$globalKey] = array_key_exists( $globalKey, $GLOBALS ) ? $GLOBALS[$globalKey] : null;
```

Comment:
Fix warnings for items in `globalsToWrap` that don't exist in `$GLOBALS`.  Setting to `null` since that's what it would have happened with the old behavior.